src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -420,3 +419,26 @@ def getPerformanceParameters(self, defaultParams):
         sizePerEvent = defaultParams.get('sizePerEvent', None) or 0
         memory = defaultParams.get('memoryRequirement', None) or 0
         return timePerEvent, sizePerEvent, memory
+    
+    def checkCreateFailConditoin(self, failedJob, failedReason):
+
+        valueMap = {"MaxWallTimeHours": "estimatedJobTime", 
+                    "MaxMemory": "estimatedMemoryUsage", 
```

Comment:
In HTCondor submit files,

foo = 1

is a "submit file command".  It has certain macro expansion rules and can be manipulated into several attributes in the resulting classad.

+foo = 1

is a classad attribute - copied directly from the submit file (without the "+") to the ad.  No macro expansion is applied.

In the case of "request_disk", it is relatively straightforward converted to the attribute "RequestDisk".  The following two lines should be equivalent in a submit file:

request_disk = 1234
+RequestDisk = 1234

However, they are *not* the same in a classad (they are different attributes).